index.md

```diff
@@ -85,6 +85,7 @@ Head over to [Scapy's GitHub Projects](https://github.com/secdev/scapy/projects)
 
 #### Slides
 
+*   [Automotive Penetration Testing with Scapy - Troopers 2019 slides](https://scapy.net/conf/troopers2019/index.html)
```

Comment:
It think that it is better to use a relative link here. That ease testing the website locally.